example_data.py

```diff
@@ -16,11 +16,8 @@ class ExampleDataLoader:
     @staticmethod
     def clean_db():
         session.flush()  # Clear out any transactions before deleting it all to avoid spurious errors.
-        engine = session.bind.engine
-        connection = engine.connect()
         for table in reversed(db.metadata.sorted_tables):
-            if engine.dialect.has_table(connection, table):
-                session.execute(table.delete())
+            session.execute(table.delete())
```

Comment:
I'm pretty sure we had the simple version of this method before, but it failed for some reason. I believe I added the 
`if engine.dialect.has_table` 
line as a solution

Reply:
Thanks @cullerton --  so that "engine.dialect.has_table"  -- is no longer available and was erroring out with a undefined error -- as if the dialect is not accessible or the has_table method was removed -- I don't remember exactly.  But I wasn't able to find a replacement, and removing the logic now passes everywhere.  So maybe we don't need it anymore?